debian/rules

```diff
@@ -7,7 +7,10 @@ override_dh_auto_build:
 	cd libs/pi_hat && cmake . && make
 	cd po && make messages.pot
 	cd po && make
-	cd touch-detect && ./build.sh
+	mkdir -p touch-detect/release
```

Comment:
I'd suggest to move all of this to `touch-detect/Makefile`

Reply:
Good idea